content/riak/kv/2.2.6/setup/installing/debian-ubuntu.md

```diff
@@ -105,28 +105,28 @@ for the target platform:
 #### Ubuntu Bionic Beaver (18.04)
 
 ```bash
-wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic/riak_2.2.6-1_amd64.deb
+wget https://files.tiot.jp/riak/kv/2.2/2.2.6/ubuntu/bionic_64/riak_2.2.6-1_amd64.deb
```

Comment:
bionic64

Reply:
that's weird - I went through and corrected tham all. Nevertheless, will accept your identical changes

Reply:
Oh, just realised you didn't make changes. I have since re-edited the file